ios/PurchasesHybridCommon/PurchasesHybridCommonSwift/OfferingsExtensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  OfferingsExtensions.swift
+//  PurchasesHybridCommonSwift
+//
+//  Created by Andrés Boedo on 4/13/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import Purchases
+
+@objc public extension Purchases.Offerings {
+
+    @objc var dictionary: [String: Any] {
+        return [
+            "all": all.mapValues { $0.dictionary },
+            "current": current?.dictionary ?? "<null>"
```

Comment:
just went back to try this out on previous versions. 

We weren't actually checking nullability at all, instead we were assigning nil. So I naively thought it might crash, since [that's what the docs say would happen](https://developer.apple.com/documentation/foundation/nsmutabledictionary/1411616-setobject?language=objc). 

> Raises an [NSInvalidArgumentException](dash-apple-api://load?request_key=lc/documentation/foundation/nsinvalidargumentexception) if anObject is nil. If you need to represent a nil value in the dictionary, use [NSNull](dash-apple-api://load?request_key=lc/documentation/foundation/nsnull).

It actually doesn't crash and instead no-ops. 
So I'll update this to explicitly not set the key if it's not present.